treeherder/webapp/api/performance_data.py

```diff
@@ -171,6 +171,10 @@ def list(self, request, project):
                                              'signatures, push_id, or '
                                              'job_id')
 
+        for id in job_ids:
+            if not(type(id) == int):
+                raise ValueError('Invalid Job ID')
```

Comment:
The logic to detect the bad values is good, but we want to return an actual response, not just raise an error (that's what this bug was complaining about). Also we could express the check a bit more concisely. Try this:

```py
if not all([type(id)==int for id in job_ids]):
    return Response('Job id(s) ,must be specified as integers', status=HTTP_400_BAD_REQUEST)
```

Could you also write a quick unit test in `tests/webapp/api/test_performance_data_api.py` to verify that we get the error as expected. You can see some examples of us testing the http api for bad input in `tests/webapp/api/test_job_details_api.py`